src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java

```diff
@@ -1240,7 +1243,15 @@ private void updateToolFlag(@NotNull final String tool, final int required)
      */
     private int getMostEfficientTool(@NotNull final Block target)
     {
-        final String tool = target.getHarvestTool(target.getDefaultState());
+        final String tool;
+        if (Compatibility.isSlimeBlock(target) || Compatibility.isSlimeLeaf(target))
+        {
+            tool = "axe";
```

Comment:
we may later want to have a compatibility proxy function for this but fine for now